### PR TITLE
ANS-9651 added basic authorization for requests

### PR DESCRIPTION
**Pull request by ropczan**

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('answear_econt');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->scalarNode('user')->cannotBeEmpty()->end()
+                ->scalarNode('password')->cannotBeEmpty()->end()
```
Comment (malarzm) [inline src/DependencyInjection/Configuration.php:19]: Technically it is a BC break, but given everything the bundle provided will not be working anyway let's merge this as is and release as next minor normally

File: tests/Unit/ConfigProviderTest.php [hunk 2]
```diff
@@ -14,9 +14,13 @@ class ConfigProviderTest extends TestCase
      */
     public function gettersAreValid(): void
     {
-        $configuration = new ConfigProvider();
+        $configuration = new ConfigProvider('test', 'Qwerty123!');
 
         $this->assertSame('http://ee.econt.com/', $configuration->getUrl());
         $this->assertSame('/services/Nomenclatures/', $configuration->getServiceURI());
+        $this->assertSame([
```
Comment (malarzm) [inline tests/Unit/ConfigProviderTest.php]: formatting looks off, the array should not be on same line as `(`

**Review by malarzm — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -12,6 +12,17 @@ composer require answear/econt-pickup-point-bundle
...
 should be added automatically to your `config/bundles.php` file by Symfony Flex.
 
+## Setup
+
+```yaml
+# config/packages/answear_econt.yaml
+answear_econt:
+    user: 'username'
+    password: 'password'
+```
```
Comment (lukasz-falda) [inline README.md:22]: sad that ECONT has no authorizationKey instead of a password :(

**Review by lukasz-falda — APPROVED** [review]

File: tests/Unit/ConfigProviderTest.php (hunk 2 above)
Comment (ropczan) [reply tests/Unit/ConfigProviderTest.php]: fixed

**Pull request merged**